tests/index.html

```diff
@@ -21,6 +21,8 @@
   WCT.loadSuites([
     'js/reactions.js',
     'js/registry.js',
+    // the next one is in a HTML file because it requires the template polyfill to be loaded
+    'js/registry-upgrade.html',
```

Comment:
The top-level `js` and `html` test directories distinction is just left over from the layout before. I don't think those two were named in relation to what they were testing in any way; they seem to have originally been about grouping the tests by the format of the tests themselves. So, this test would make sense to go in `html`.

(FWIW, I'd be up for reorganizing these folders by what they're testing at some point, rather than what kind of tests they are.)

Reply:
Feel free to push this branch to move the file. Else I can get to it tomorrow morning.

Reply:
Ok, I'll make the update.